### ISSUE TITLE
Sabetha Markers

### ISSUE THREAD

**Issue by MohAl-s** [QuitarHero/Hero-Timers#4]
Any chance we can get the old arrow, circle, heart and square or have it added as a selectable option for the cannons on sabetha?

**Comment by QuitarHero** [comment]
If the Timers Module UI was significantly more compressed so many other timers can be shown at once, filtered better, and provide me more customization as a timer-creator, I would feel far better about maintaining utilitarian options such as commander icons on warnings, as well as bring back various timers. But since the module developer has been inactive for at least a year, it doesn't feel like good practice to provide what I used to in the past, such as the 9 Cairn timers, 5 Sabetha timers, 3 Matthias timers (or in short more gimmicky / specific use-case timers), when that all and more should be dealt with on the module side.

Perhaps more importantly, I made the changes to Sabetha, Dhuum, and more so they would be in line with how the Strike timers are developed; in-world markers when relevant, actual icons for encounter mechanics, and countdowns to "Next: mechanic" to prepare players for incoming mechanics.

In short, I will not be bringing back commander icons back to Sabetha timers in this case, but also Dhuum, and anywhere else, at least until the module is in a far more functional and user-friendly state.

You're more than welcome to change them back yourself. Extract the `Hero.Timer.Pack.zip` into the same folder it's in, the `/timers` directory, delete the `.zip`, open the relevant `.bhtimer` files in a text editor, then change the texture to whatever icon you'd like found in the extracted `/Data` folder. I forgot to remove the commander icons for the current release, so they're still there to reference.

**Comment by MohAl-s** [comment]
Thanks for the reply and the suggestion, I'll try to change them back.
